### PR TITLE
Localization stable-4.5 string update

### PR DESCRIPTION
**Pull request by rooftopcellist**
Localization Update from 4-19 push.  Add Korean translated .po file.

### CODE REVIEW

**Comment by rooftopcellist** [comment]
This PR has all of the changes https://github.com/ansible/ansible-hub-ui/pull/2030 does and more, so the old PR can be closed now.

**Comment by rooftopcellist** [comment]
cc @himdel for review

**Review by newswangerd — CHANGES_REQUESTED** [review]
Can you open this against master instead?

File: locale/zh.po [hunk 1]
```diff
@@ -4,29 +4,28 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: @lingui/cli\n"
-"Language: zh\n"
+"Language: en\n"
```
[the remark this answers is not in the document]
Comment (rooftopcellist) [reply locale/zh.po]: @himdel looks like it doesn't show up in the compiled .js files, so it's probably not an issue, but better to be safe than sorry : )  Nice catch.

**Comment by himdel** [comment]
Manual backport in https://github.com/ansible/ansible-hub-ui/pull/2041 :)

**Comment by rooftopcellist** [comment]
Thanks for doing that @himdel !

**Pull request merged**